### PR TITLE
Transition

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
r?

File: components/amount-buttons.jsx [hunk 1]
```diff
@@ -45,6 +51,30 @@ var AmountButtons = React.createClass({
 
     updateAmountOptions($.urlParam(AMOUNT_SET_PARAM));
 
+    $("#amount-other-input")[0].addEventListener("input", function() {
```
Comment (alicoding) [inline components/amount-buttons.jsx]: I don't know how I feel about this `[0]`

File: pages/sequential.jsx [hunk 2]
```diff
@@ -60,38 +66,22 @@ var Sequential = React.createClass({
       }
     });
 
-    function calculateHeight() {
-      if (!win.location.hash) {
-        return;
-      }
-      $('.sequence-page-container').height($(win.location.hash).height());
-    }
+    var calculateHeight = this.calculateHeight;
```
Comment (alicoding) [inline pages/sequential.jsx]: interesting!

**Comment by alicoding** [comment]
you need to rebase

**Comment by ScottDowne** [comment]
You're a rebase!

File: components/amount-buttons.jsx (hunk 1 above)
Comment (ScottDowne) [inline components/amount-buttons.jsx]: Wanna talk about it?

**Comment by ScottDowne** [comment]
Done rebasing.

File: pages/sequential.jsx [hunk 3]
```diff
@@ -429,8 +369,13 @@ var Sequential = React.createClass({
             var formData = {};
             $theForm.serializeArray().map(function(x){formData[x.name] = x.value;});
             var transaction = {
+<<<<<<< HEAD
```
Comment (alicoding) [inline pages/sequential.jsx]: ?

Comment (ScottDowne) [inline pages/sequential.jsx]: Yeah, it's new syntax, es7 awesomeness.

Super arrow head notation.

**Comment by ScottDowne** [comment]
updated

File: pages/sequential.jsx [hunk 4]
```diff
@@ -170,32 +157,14 @@ var Sequential = React.createClass({
...
 
-    $theForm.find("[name='donation_amount_other']").keyup(function() {
-      updateDonateButtonText($(this).val());
-    });
-    $theForm.find("[name='donation_amount']").change(function() {
-      $theForm.find("[name='donation_amount_other']").val("");
-    });
-    $theForm.find("[name='donation_amount_other']").keydown(function(event) {
-      var functionKeys = [8, 9, 13, 27, 37, 39, 46, 110, 190]; // backspace, tab, enter, escape, left arrow, right arrow, delete, decimal point, period
-      var numberKeys = [48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105]; // numbers
```
Comment (alicoding) [inline pages/sequential.jsx]: We don't need all this now??

Comment (alicoding) [inline pages/sequential.jsx]: Ah I see you've moved them

**Comment by alicoding** [comment]
R+

**Pull request merged**